docs/conf.py

```diff
@@ -221,7 +221,7 @@
 # Generate list of package index
 providers_packages_roots = {
     name.rpartition("/")[0]
-    for entity in ["hooks", "operators", "sensors"]
+    for entity in ["creds", "hooks", "operators", "sensors"]
```

Comment:
I think, you should also update https://airflow.readthedocs.io/en/latest/_api/index.html and https://airflow.readthedocs.io/en/latest/integration.html

Reply:
thank you @mik-laj passing now